credentials/tls/certprovider/meshca/plugin.go

```diff
@@ -149,9 +149,6 @@ func (p *providerPlugin) updateKeyMaterial(ctx context.Context) time.Duration {
 			select {
```

Comment:
Just noticed the `select` above this one (github won't let me comment on it directly).  This can be simplified as: `if ctx.Err() == nil { return 0 }`